revelatormodsystem.py

```diff
@@ -2,49 +2,67 @@
 import sys
 import os
 
-scriptextract = False
+script_extract_folder = ''

```

Comment:
Changed the boolean into a string, if the path of the script folder is set it means that the argument was given.